test/e2e/scenario/rum/sessions.scenario.ts

```diff
@@ -115,4 +116,26 @@ describe('rum sessions', () => {
         expect(intakeRegistry.replaySegments.length).toBe(1)
       })
   })
+
+  describe('third party cookie clearing', () => {
+    createTest('after a 3rd party clears the cookies, stop the session')
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I am wondering if this test was not passing before those changes 🤔
FMU, what we want to achieve with this change is not to stop the session but to avoid to create new ones, it is probably what we should test.